dask/layers.py

```diff
@@ -0,0 +1,194 @@
+import tlz as toolz
+from importlib import import_module
+
+from .core import keys_in_tasks
+from .utils import insert, stringify, stringify_collection_keys
+from .highlevelgraph import Layer
+
+
+def run_ext_function(func, *args):
+    if isinstance(func, dict):
+        func = getattr(import_module(func["__module__"]), func["__name__"])
+    return func(*args)
```

Comment:
I personally prefer this variety of the function over the pickle version, but curious to know what others think!